1es-azure-pipeline.yml

```diff
@@ -54,6 +54,13 @@ parameters:
   type: string
   default: Test
   values: [ 'Test', 'Real' ]
+- name: PRERELEASE
+  displayName: Prerelease
+  type: boolean
+  default: false
+  values:
+  - true
+  - false
```

Comment:
When type is `boolean`, I don't know if `values` does anything here...

Reply:
so trueee, fixed, thanks